vector/src/agent/java/fr/gouv/tchap/config/TargetConfiguration.kt

```diff
@@ -20,3 +20,5 @@ package fr.gouv.tchap.config
 // ex: const val ENABLE_FEATURE1 = true
 
 const val CONFIGURATION_VARIANT = "agent"
+
+const val ENABLE_ROOM_RETENTION = false
```

Comment:
ENABLE_ROOM_RETENTION_CHANGE

Reply:
FYI I will not change this const name because it will be used finally to hide all the retention stuff (not only the edition)